src/app/components/acknowledge-response/acknowledge-response.component.scss

```diff
@@ -1,3 +1,5 @@
+@import "src/assets/styles/_variables.scss";
```

Comment:
background: #020202a1;
Replace the background color as well with variables.scss color